src/components/navigation/AppNavigationBoard.vue

```diff
@@ -253,6 +253,9 @@ export default {
 			try {
 				const newBoard = await this.$store.dispatch('cloneBoard', this.board)
 				this.loading = false
+				if (newBoard instanceof Error) {
+					throw newBoard
+				}
```

Comment:
Can we also make sure that we hide the clone menu entry if that permission is not available?